packages/lab/src/stepped-tracker/Step.Connector.tsx

```diff
@@ -0,0 +1,28 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+
+import stepConnectorCSS from "./Step.Connector.css";
+
+export namespace StepConnector {
+  export interface Props {
+    className?: string;
+  }
+}
+
+const withBaseName = makePrefixer("saltStepConnector");
+
+export function StepConnector({ className }: StepConnector.Props) {
```

Comment:
All components should accept HTML attributes

Reply:
this is an internal only component, don't think it is necessary at this point, actually I might remove the className as well